js/plugins/grid.html

```diff
@@ -79,7 +79,7 @@
             var shortcodes = '[bs_row class="row"]';
             $('#demo_grid > div').each(function() {
                 $(this).removeClass('demo_col active');
-                var clss = $(this).attr('class');
+                var clss = $(this).attr('class').replace(/xs/g, "sm");
```

Comment:
Ah, *that* makes sense! However, I think it might be a little misleading for the output of the visual editor to differ what from the user just created.

This line was removed before I merged your changes, but I would be happy to consider an alternative solution. Perhaps an "improved" visual editor which makes it more clear what is going to go in the page, or gives the user the real-estate needed to see those `sm` breakpoints without needing to swap them out after-the-fact.

Thanks again for submitting these changes @emRae